AutomatedTesting/Gem/Code/Source/AutoGen/NetworkTestPlayerComponent.AutoComponent.xml

```diff
@@ -13,4 +13,29 @@
     <NetworkInput Type="float" Name="FwdBack" Init="0.0f" ExposeToScript="true"/>
     <NetworkInput Type="float" Name="LeftRight" Init="0.0f" ExposeToScript="true"/>
 
+    <RemoteProcedure Name="AutonToAuth" InvokeFrom="Autonomous" HandleOn="Authority" IsPublic="false" IsReliable="false" GenerateEventBindings="true" Description="" >
+        <Param Type="float" Name="SomeFloat" />
+    </RemoteProcedure>
+  
+    <RemoteProcedure Name="AutonToAuthNoParams" InvokeFrom="Autonomous" HandleOn="Authority" IsPublic="false" IsReliable="false" GenerateEventBindings="true" Description="" />
+
+    <RemoteProcedure Name="AuthToAuton" InvokeFrom="Authority" HandleOn="Autonomous" IsPublic="false" IsReliable="false" GenerateEventBindings="true" Description="" >
+        <Param Type="float" Name="SomeFloat" />
+    </RemoteProcedure>
+    
+    <RemoteProcedure Name="AuthToAutonNoParams" InvokeFrom="Authority" HandleOn="Autonomous" IsPublic="false" IsReliable="false" GenerateEventBindings="true" Description="" />
```

Comment:
Why not spell out the full name of Autonomous instead of 'Auton'? Etc.

Reply:
fixed!